tests/test_hvac_csc.py

```diff
@@ -136,6 +87,17 @@ async def test_version(self):
     async def test_bin_script(self):
         await self.check_bin_script(name="HVAC", index=None, exe_name="run_hvac.py")
 
+    async def _verify_evt_deviceEnabled(self, subsystem):
+        # Default mask indicating that the devices, that always are enabled,
```

Comment:
I find the comma before "that always are" confusing. Do you mean "Mask indicating the devices that should be enabled"?

Reply:
No, there are three devices that always are enabled. I reformulated the phrase a bit.